Documenting roles, responsibilities and recruitment process (#39)
LibraryCarpentry/governance: succession.md

# Library Carpentry Advisory Committee Succession Planning

> This document outline processes for recruiting new members to the Library Carpentry Advisory Committee and activities required as part of onboarding and offboarding.

Reviewer: ```suggestion
This document outlines processes for recruiting new members to the Library Carpentry Advisory Committee and activities required as part of onboarding and offboarding.
```

## Process for membership recruitment checklist 
- Co-Chairs lead the recruitment process
- Call for current members to indicate if they intend to relinquish roles
- Identify number of members to replace

> - Draft EOI form [](using this template) 

Reviewer: did you mean to add a link here?

Author: Yes! Thanks for picking this up.

> - Draft EOI form [https://docs.google.com/forms/d/11bkkHkQc1PbX5ez7frMtWLGPLWtQWBjmLrU3BGr7k4c/edit](using this template) 

Reviewer 2: Might clarify to spell out "Expression of Interest (EOI) form" on this first mention.

Author: Thanks for picking that up! I'll submit an update. I havne't figured out how to make changes within the github review process.

- Update blog post for call out and send to The Carpentries [](using this template)
- Distribute EOI form with a reasonable turnaround
- LCAG members assess the applications using the [](rubric)
- If more applications than available seats, use the rubric scoring to identify the shortlist
- Contact successful applicants and obtain info for LCAG website page
- Open nominations for members to volunteer for Co-chairs or Secretary roles. If more nominations than roles available is received, a vote will be taken.

## Onboarding

> - New members provide bio information for Advisory Group page, see e.g. [Advisory Group webpage](https://librarycarpentry.org/advisory/) 

Reviewer: ```suggestion
- New members provide biography information for Advisory Group page, see e.g. [Advisory Group webpage](https://librarycarpentry.org/advisory/) 
```

- Agree to attend at least 50-75% of the LC Advisory Group meetings and relay if you have meeting time constraints
- Review [Roles and Responsibilities]()
- New members encouraged to watch at least one LC lesson and consider joining the Maintainer team for that lesson, which provides context from lesson development (e.g. new or under development)
- Help new members connect with all communication channels, TopicBox (LC Advisory Group, Libraries), Slack (#libraries), Twitter, GitHub Governance repository, LC Advisory Group Etherpad, Google community calendar
- Review Etherpad of previous LC Advisory Group meeting agendas/notes
- Review Governance repo, review closed issues (from combined meeting), decide if issues should be reopened, new issues should be created, or abandoned
- Bring issues/new ideas to prioritizing meeting, where members vote on the top 5-7 items to address in the next 1-2 years

> - Joint blog post with outgoing members, new members provide brief summary of where they are coming, why they want to be a part of the Advisory Group, and what they might want to work on moving forward.

Reviewer: ```suggestion
- Joint blog post with outgoing members, new members provide brief summary of where they are coming from, why they want to be a part of the Advisory Group, and what they might want to work on moving forward.
```

## Offboarding
- Help with recruiting (replacements)
- Identify leads on projects/issues opened 

> - By default close/sunset projects/issues, provide description/outcomes, present closed to issues to new members who can then reopen, create new issues, or abandon

Reviewer: ```suggestion
- By default close/sunset projects/issues, provide description/outcomes, present closed issues to new members who can then reopen, create new issues, or abandon
```

> - Be present at least a combined meeting where outgoing members can explain context of projects/issues to incoming members; identify at least one responsible person that will be available for follow up question

Reviewer: ```suggestion
- Be present at at least a combined meeting where outgoing members can explain context of projects/issues to incoming members; identify at least one responsible person that will be available for follow up question
```

> - Joint blog post of welcoming new members, outgoing members, co-chairs responsible for spearheading blog post, individuals should provide summaries of what they did/would like to do (what interested them in joining). Outgoing members thanked for their service.

Reviewer: ```suggestion
- Joint blog post to welcome new members, list and thank outgoing members. Co-chairs are responsible for spearheading the blog post, individuals should provide summaries of what they did/would like to do (what interested them in joining). Outgoing members thanked for their service.
```
